test/Microsoft.Azure.WebJobs.Host.UnitTests/AttributeClonerTests.cs

```diff
@@ -64,6 +65,13 @@ public class Attr4: Attribute
             [AutoResolve]
             public string AutoResolve { get; set; }
         }
+        
+        // Test with DefaultValue.MemberName
+        public class Attr5 : Attribute
+        {
+            [AutoResolve(Default = DefaultResolveValue.MemberName)]
```

Comment:
For the purposes of bindings, the member will ALWAYS be a method name, right? Your code even hard codes to method internally. E.g. we won't every use this for properties, events, etc. So I think we should rename to MethodName?